.github/workflows/docker.yml

```diff
@@ -24,16 +24,14 @@ on:
       - .github/workflows/docker.yml
     branches:
       - master
-    tags:
-      - v[12].[0-9]+.[0-9]+
```

Comment:
Hopefully it's OK to remove these, I couldn't think of a reason or a way to push a tag through a pull request.